localization/ndt_scan_matcher/include/ndt_scan_matcher/ndt_scan_matcher_core.hpp

```diff
@@ -145,6 +145,10 @@ class NDTScanMatcher : public rclcpp::Node
     const pclomp::NdtResult & ndt_result, const Eigen::Matrix4f & initial_pose_matrix,
     const rclcpp::Time & sensor_ros_time);
 
+  pcl::PointCloud<pcl::PointXYZRGB>::Ptr visualize_point_score(
+    const pcl::shared_ptr<pcl::PointCloud<PointSource>> & sensor_points_in_map_ptr,
+    const float & lower_nvs, const float & upper_nvs);
```

Comment:
I like that this function returns `pcl::PointCloud<pcl::PointXYZRGB>` instead of `pcl::PointCloud<pcl::PointXYZRGB>::Ptr`.